gst.so/CMakeLists.txt

```diff
@@ -82,11 +82,12 @@ add_compile_options(-DRVS_ROCBLAS_VERSION_FLAT=${RVS_ROCBLAS_VERSION_FLAT})
 # Determine HSA_PATH
 if(NOT DEFINED HIPCC_PATH)
   if(NOT DEFINED ENV{HIPCC_PATH})
-    set(HIPCC_PATH "${ROCM_PATH}/hip" CACHE PATH "Path to which hipcc runtime has been installed")
+    set(HIPCC_PATH "${ROCM_PATH}" CACHE PATH "Path to which hipcc runtime has been installed")
      else()
        set(HIPCC_PATH $ENV{HIPCC_PATH} CACHE PATH "Path to which hipcc runtime has been installed")
      endif()
 endif()
+message(STATUS "HIPCC PATH Set as: ${HIPCC_PATH}" )
```

Comment:
Please remove this status message log.

Reply:
These were added to print the flags giving more info for the build. As per comments, removing this prints in the latest